spec/base_spec.rb

```diff
@@ -426,6 +427,28 @@ class DummyReplacedModel < DummyElasticSearchModel; end
         end
       end
 
+      context "new and save" do
+        it "supports calling new and then saving later" do
+          dummy_model = DummyElasticSearchModel.new(my_string: "Hello")
+
+          # Not saved yet
+          refresh_index
+          search_response = @elasticsearch_test_client.search(index: DummyElasticSearchModel.index_name)
+          expect(search_response.dig("hits", "total")).to eq(0)
+
+          dummy_model.save!
+          search_hit = refresh_and_find_search_hit
+          expect(search_hit["_index"]).to eq(dummy_model.index_name)
+          expect(search_hit["_source"]["rehydration_class"]).to eq(dummy_model.type)
+        end
+
+        it "does not allow save to be called on a model that has already been updated" do
```

Comment:
It would be good to also have a test for a model fetched and rehydrated from Elasticsearch.

Reply:
:+1: